javascript-source/core/updates.js

```diff
@@ -877,9 +877,9 @@
         $.inidb.set('updates', 'installedv3.0.1', 'true');
     }
 
-    /* version 3.1.3 updates */
-    if (!$.inidb.exists('updates', 'installedv3.1.3') || $.inidb.get('updates', 'installedv3.1.3') != 'true') {
-        $.consoleLn('Starting PhantomBot update 3.1.3 updates...');
+    /* version 3.2 updates */
```

Comment:
3.2.0

https://semver.org/

Reply:
Yeah, that slipped through. I know what semantic versioning is… Guess I've missed 3.2.0, so what will be the next version I should aim for for this feature?